package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "axios": "^1.6.8",
     "next": "14.2.1",
+    "node": "^22.6.0",
```

Comment:
it looks like you are still adding node to the project here. you need to remove this line from this file, and run `npm install` again

Reply:
I did that but it didn't re-install node